fastlane/Fastfile

```diff
@@ -539,6 +533,29 @@ platform :android do
     )
   end
 
+  lane :create_backmerge_pr do |version: release_version_current|
+    create_release_backmerge_pull_request(
+      repository: GH_REPOSITORY,
+      source_branch: "release/#{version}",
+      default_branch: DEFAULT_BRANCH,
+      labels: ['Releases'],
+      milestone_title: release_version_next
+    )
+  rescue StandardError => e
+    error_message = <<-MESSAGE
+      Error creating backmerge pull request:
+
+      #{e.message}
+
+      If this is not the first time you are running the release task, the backmerge PR for the version `#{version}` might have already been previously created.
+      Please close any previous backmerge PR for `#{version}`, delete the previous merge branch, then run the release task again.
+    MESSAGE
+
+    buildkite_annotate(style: 'error', context: 'error-creating-backmerge', message: error_message) if is_ci
+
+    UI.user_error!(error_message)
```

Comment:
I just realized that I copy-pasted this error handling from the Fastfile of another repo… which already had the lanes being executed by CI instead of developers' machines.

Moving PCAndroid to have the fastlane lanes being run on CI is the next step in my work on improving Release Automation, but it's not implemented yet. Which means that in the meantime, this `rescue` block in case an error occurs creating the PR: (1) not only provides a slightly misleading error message / recovery suggestion in the meantime and (2) will likely fail when it'll try to call `buildkite_annotate` to annotate the CI build with the error message… since for the time being that lane is executed on devs' Macs not on CI.

Also, [now that our `create_release_backmerge_pull_request` action from `release-toolkit` has been improved to auto-delete past intermediate branch of past backmerge PRs before creating a more recent one](https://github.com/wordpress-mobile/release-toolkit/pull/601/files), thus making this error use case less likely if not impossible… I'm not sure the error handling and recovery suggestion are necessary to be added in our `Fastfiles` anymore 🤔 (cc @iangmaia @spencertransier thoughts on that last part?)

Reply:
Update: removed the `rescue` block in 71668bd31

Reply:
Ah that's a good point 👍 
I mean, there can still be errors in the process but I guess we're fine just letting them propagate -- this specific handling was indeed for the "backmerge already existing" case.